recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ test:
         - unxz --help
         - lzma --help
         - conda inspect linkages -n _test xz  # [linux]
+        - otool -L $PREFIX/lib/liblzma.5.dylib  # [osx]
```

Comment:
This is not really needed, but I must be certain that we won't get

```
Reason: Incompatible library version: libtiff.5.dylib requires version 8.0.0 or later,
but liblzma.5.dylib provides version 6.0.0
```

on OS X.